.buildkite/pipeline.yml

```diff
@@ -0,0 +1,25 @@
+steps:
+  - label: ':python::hammer: tox build, test, and release'
+    if: build.branch == "master" && pipeline.slug == "yukon-release"
+    id: toxic-docker-release
+    command: "./.buildkite/release.sh"
+    artifact_paths: 
+      - ".tox/report/tmp/*"
+      - ".tox/mypy/tmp/*"
+      - ".tox/flake8/tmp/*"
+      - ".tox/dist/*.zip"
+      - ".tox/py*-test/tmp/xunit-result.xml"
+    plugins:
+      - docker#v3.3.0:
+          workdir: /repo
+          image: "uavcan/toxic:py35-py38-sq"
+          propagate-environment: true
+          environment:
+            - "SONARQUBE_TOKEN"
+            - "PYPI_PASSWORD"
+            - "BUILDKITE"
+            - "BUILDKITE_BUILD_NUMBER"
+    agents:
+      queue: 'default'
+    timeout_in_minutes: 15
+  # TODO: add verification builds
```

Comment:
You should add the verification builds in the Buildkite UI itself (see https://buildkite.com/uavcan/nunavut-pr/settings). The reason is any third party can initiate a verification build by submitting a PR to our organization's repo. If you pull the verification build steps from source then the committer can easily steal secrets and take over your CI possibly gaining access to your source repositories.

So, remove this TODO from here.

Reply:
Removed